src/Core/DTO.fs

```diff
@@ -237,7 +237,7 @@ module DTO =
           References : ProjectReferencePath array
           Logs : Map<string, string>
           OutputType : string
-          Info : ProjectResponseInfoDotnetSdk
+          Info : {| Data: ProjectResponseInfoDotnetSdk |}
```

Comment:
not sure if we intended for the data structure to change here, but this was causing several problems in various places when trying to use project data.